snuba/cli/config.py

```diff
@@ -0,0 +1,95 @@
+import click
+import json
+
+from snuba import state
+
+
+def human_fmt(values):
+    lines = []
+    for k, v in values.items():
+        lines.append(f"{k} = {repr(v)} ({type(v).__name__})")
+    return "\n".join(lines)
+
+
+def json_fmt(values):
+    return json.dumps(values)
+
+
+def get_user():
+    import getpass
+
+    return f"{getpass.getuser()} (cli)"
```

Comment:
Not sure if you all have an opinion on this, but I wanted the user to reflect the command line user, but also differentiate itself since we use emails on the API. So I added this (cli) thing to indicate it's from the CLI, but I can do whatever you want.